conda/core/subdir_data.py

```diff
@@ -322,7 +344,7 @@ def _pickle_me(self):
 
     def _read_local_repdata(self, etag, mod_stamp):
         # first try reading pickled data
-        _pickled_state = self._read_pickled(etag, mod_stamp)
+        _pickled_state = False  # self._read_pickled(etag, mod_stamp)
```

Comment:
We can change `REPODATA_PICKLE_VERSION` in this module to fix the loads-wrong-pickle problem. However we don't like having to spend the time to save the pickle in the first place. Whether the pickle is worth it could depend on how often you run conda versus how often the repository gets updated.

Reply:
Also, if we pickle the un-parsed repodata we haven't saved the user their repodata -> PackageRecord time on the next run.

If we wanted to get complex, we could do the deferred parsing during another download.

Reply:
> Whether the pickle is worth it could depend on how often you run conda versus how often the repository gets updated.

My best guess is that conda commands are often run in short succession until an environment is set up correctly, and then not updated until a reason comes along.

> If we wanted to get complex, we could do the deferred parsing during another download.

If possible, we should simplify, unless it loses us correctness.